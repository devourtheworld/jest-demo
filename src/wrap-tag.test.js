import wrapTag from './wrap-tag.js';

test('can wrap tag', () => {
    const wrappedCode = wrapTag('Hello World', 'h1');
    expect(wrappedCode).toMatchSnapshot();
}) 
