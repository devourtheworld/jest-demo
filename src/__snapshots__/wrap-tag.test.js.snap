// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can wrap tag 1`] = `"<h1>Hello World</h1>"`;
